polygon-digital-carbon/src/TransferHandler.ts

```diff
@@ -121,6 +128,11 @@ export function handleToucanPuroDetokenizationFinalized(): void {}
 
 export function handleToucanPuroDetokenizationReverted(): void {}
 
+export function handleCCO2Retired(event: burnedCO2Token): void {
+  if (event.params.amount == ZERO_BI) return
```

Comment:
seems we have this zero handling in two places unnecessarily https://github.com/KlimaDAO/klima-subgraph/pull/175/files#diff-fa10acc11e4249bed4407adc2e6dc9d28b10cfedb4adb8deaa1bbd4e36706a50R231